src/components/dataset/details/DatasetDetailView.jsx

```diff
@@ -1,81 +1,105 @@
 import { connect } from 'react-redux';
 import { withStyles } from '@material-ui/core/styles';
 import { getDatasetById, getDatasetPolicy } from 'actions/index';
+import { Grid, Typography } from '@material-ui/core';
 import DatasetInfoCard from './DatasetInfoCard';
 import CreateFullSnapshotView from './CreateFullSnapshotView';
+import DatasetRelationshipsPanel from './VisualizeRelationshipsPanel';
 
 const styles = (theme) => ({
   root: {
     // TODO: expect this to change as more components are added
-    padding: theme.spacing(4),
-    width: '70%',
-    margin: 'auto',
+    height: '100%',
+    display: 'grid',
+    gridTemplateColumns: '1fr 2fr',
+    margin: '1rem 0.5rem',
   },
   headerText: {
     fontWeight: theme.typography.bold,
     textTransform: 'uppercase',
-    margin: `${theme.spacing(2)}px ${theme.spacing(1)}px`,
+  },
+  infoColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  infoColumnPanel: {
+    flexGrow: 1,
+  },
+  mainColumn: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: '0.5rem',
+  },
+  mainColumnRow1: {},
+  mainColumnRow2: {},
+  mainColumnRow3: {
+    flexGrow: 1,
+  },
+  spacer: {
+    height: 64,
   },
 });
 
-class DatasetDetailView extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      creatingSnapshot: false,
-    };
-  }
-
-  static propTypes = {
-    classes: PropTypes.object,
-    dataset: PropTypes.object,
-    datasetPolicies: PropTypes.array,
-    dispatch: PropTypes.func.isRequired,
-    match: PropTypes.object,
-  };
+const DatasetDetailView = ({
+  classes,
+  dataset,
+  datasetPolicies,
+  dispatch,
+  match: {
+    params: { uuid },
+  },
+}) => {
+  const [creatingSnapshot, setCreatingSnapshot] = useState(false);
 
-  componentDidMount() {
-    const { match, dispatch } = this.props;
-    const { uuid } = match.params;
+  useEffect(() => {
     dispatch(getDatasetById(uuid));
     dispatch(getDatasetPolicy(uuid));
-  }
-
-  openSnapshotCreation = (isOpen) => {
-    this.setState({ creatingSnapshot: isOpen });
-  };
+  }, []);
 
-  render() {
-    const { classes, dataset, datasetPolicies, match } = this.props;
-    const { creatingSnapshot } = this.state;
-    const { uuid } = match.params;
-    if (dataset && datasetPolicies && dataset.id === uuid) {
-      return (
-        <div className={classes.root}>
-          <div className={classes.headerText}>Dataset Information</div>
-          <DatasetInfoCard
-            dataset={dataset}
-            datasetPolicies={datasetPolicies}
-            openSnapshotCreation={this.openSnapshotCreation}
-          />
-          <CreateFullSnapshotView
-            open={creatingSnapshot}
-            openSnapshotCreation={this.openSnapshotCreation}
-          />
+  return datasetPolicies && dataset && dataset.id === uuid ? (
+    <Fragment>
+      <h2 style={{ margin: '0.5rem' }}>{dataset.name}</h2>
+      <div className={classes.root}>
```

Comment:
Why not use `Grid` components here?  I've found those make the pseudo-HTML a lot more readable since you can clearly see what the layout elements are.